test/fib-dbg.wasm.fromBinary

```diff
@@ -72,7 +72,9 @@
    (return
     (local.get $1)
    )
+   (unreachable)
   )
+  (unreachable)
```

Comment:
Are these extra instructions expected?

Reply:
Yes. The binary parser no longer skips ahead past unreachable instructions, so whereas the old code would have skipped the `unreachable`s after parsing the `return` and the unreachable block, the new code parses everything.

Reply:
The old parser would note that the block containing the return also finalizes to unreachable and would skip parsing that second `unreachable` as well.